.github/workflows/cri_test.yml

```diff
@@ -68,3 +68,35 @@ jobs:
     - name: Cleaning
       if: ${{ always() }}
       run: ./scripts/github_runner/clean_cri_runner.sh
+  
+  gvisor-cri-tests:
+    name: gVisor CRI tests
+    env:
+        GITHUB_RUN_ID: ${{ github.run_id }}
+        GITHUB_VHIVE_ARGS: "-dbg"
+    runs-on: ubuntu-18.04
+
+    steps:
```

Comment:
try checking out the repo ;) 
```
    - name: Check out code into the Go module directory
      uses: actions/checkout@v2

Reply:
Thanks, that helps

Reply:
Setting up kubernetes doesn't seem to work on the github server:
`[kubelet-check] The HTTP call equal to 'curl -sSL http://localhost:10248/healthz' failed with error: Get "http://localhost:10248/healthz": dial tcp [::1]:10248: connect: connection refused.`
Should I move the test to self-hosted, or can we somehow fix this issue.